### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dallen68**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,88 +1,79 @@
...
-Give an example
+mvn test
 ```
 
-### And coding style tests
-
-Explain what these tests test and why
+### End to End Test
 
+To run end to end tests run
```
Comment (josefj1519) [inline README.md:39]: Put this in deployment.  To deploy locally... do this...

File: README.md [hunk 2]
```diff
@@ -1,88 +1,79 @@
...
-* [Dropwizard](http://www.dropwizard.io/1.0.2/docs/) - The web framework used
+* [BootStrap 4](https://getbootstrap.com/docs/4.0/getting-started/introduction/) - The CSS framework used
 * [Maven](https://maven.apache.org/) - Dependency Management
-* [ROME](https://rometools.github.io/rome/) - Used to generate RSS Feeds
+* [jQuery](https://api.jquery.com/) - Javascript Library
+* [YouTube Data API](https://developers.google.com/youtube/v3) - YouTube API
 
 ## Contributing
 
 Please read [CONTRIBUTING.md](https://gist.github.com/PurpleBooth/b24679402957c63ec426) for details on our code of conduct, and the process for submitting pull requests to us.
```
Comment (josefj1519) [inline README.md]: Get rid of this for now

File: README.md [hunk 3]
```diff
@@ -1,88 +1,79 @@
...
+with the proper project ID in the pom.xml file.
 
 ## Built With
 
-* [Dropwizard](http://www.dropwizard.io/1.0.2/docs/) - The web framework used
+* [BootStrap 4](https://getbootstrap.com/docs/4.0/getting-started/introduction/) - The CSS framework used
 * [Maven](https://maven.apache.org/) - Dependency Management
-* [ROME](https://rometools.github.io/rome/) - Used to generate RSS Feeds
+* [jQuery](https://api.jquery.com/) - Javascript Library
+* [YouTube Data API](https://developers.google.com/youtube/v3) - YouTube API
```
Comment (josefj1519) [inline README.md]: YouTube Data API v3

File: README.md [hunk 4]
```diff
@@ -1,88 +1,79 @@
 # PromoTube
 
 PromoTube is a webapp that allows users to easily find promo codes and affiliate links that go with their favorite creators or products.
+It uses the YouTube Data API to find videos from a given channel. We then parse the video descriptions and return the found promo codes.
```
Comment (josefj1519) [inline README.md]: PromoTube uses the YouTube Data API to scrape promo-codes from descriptions given channel's URL or business's name.  What do you think?

File: README.md [hunk 5]
```diff
@@ -1,88 +1,78 @@
 # PromoTube
 
 PromoTube is a webapp that allows users to easily find promo codes and affiliate links that go with their favorite creators or products.
```
Comment (josefj1519) [inline README.md]: PromoTube is a browser-based web application which extracts promotional codes and affiliate links from YouTube video descriptions.

File: README.md [hunk 6]
```diff
@@ -1,88 +1,78 @@
...
+It uses the YouTube Data API to find videos from a given channel. We then parse the video descriptions and return the found promo codes.
+It also is able to find promo codes from business's in a similar way.
```
Comment (josefj1519) [inline README.md:8]: Add Background section
 ## Background
YouTube content creators receive external sponsorships from companies to advertise products to their subscribers. This often comes in the form of promo codes and affiliate links which are written in the description of their YouTube video. These provide discounts for their followers while also giving some proceeds back to the creator. Currently, there exists no way for viewers or consumers to easily aggregate these discount codes without manually searching through videos. By compiling the promotional codes from a specific content creator, users will be able to more easily support their favorite YouTubers and find discounts for themselves.

File: README.md [hunk 7]
```diff
@@ -1,88 +1,77 @@
...
+## Deployment
 
-Explain what these tests test and why
+To deploy the app on a server run
 
 ```
-Give an example
+mvn package appengine:deploy
 ```
+with the proper project ID in the pom.xml file.
```
Comment (josefj1519) [inline README.md:52]: Add somewhere that you also need Youtube api key

**Comment by margaretchan** [comment]
Note to add screenshot and link to website once fully deployed.

**Review by josefj1519 — APPROVED** [review]

**Review by margaretchan — APPROVED** [review]

**Pull request merged**